scripts/globals/survival_guide.lua

```diff
@@ -142,8 +142,8 @@ dsp.survivalGuide.onEventUpdate = function(player, csid, option)
 end
 
 dsp.survivalGuide.onEventFinish = function(player, eventId, option)
-    if cutsceneID == eventId and not (optionMap.TELEPORT > 1 and optionMap.TELEPORT_MENU <= 7)
-        and not (option == 65539)  then
+    if cutsceneID == eventId and not (option > optionMap.TELEPORT and option <= optionMap.TELEPORT_MENU)
```

Comment:
`if bit.band(option, 0xFF) == optionMap.TELEPORT then`

Reply:
I completely missed that this was doing a greater and less than check on that.

#qualityRevieweingTeo